### PR TITLE
Delete LBBackendAddressPoolWithBackendAddressesPutWithNICIPConfigurat…

### PR DESCRIPTION
**Pull request by erichrt**
…ion.json

We do not support NIC based PUTs for this REST call

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by erichrt** [comment]
@anavinahar, @ninzavivek as FYI

**Comment by MyronFanQiu** [comment]
@erichrt Hi. We cannot just delete one example without changing swagger definition.

**Comment by erichrt** [comment]
Hi @MyronFanQiu we need to delete this example as the PUT operation does not support NIC IP configurations but the schema of the Backend Address Pool does thus we are not updating the spec. I will set up a short 15 minute sync for us to discuss.

**Comment by erichrt** [comment]
@MyronFanQiu updated with latest commit

**Comment by MyronFanQiu** [comment]
@erichrt well. emmm. Looks like we cannot directly remove one example. You may need to fix this one with other changes together to pass the CI.

**Review by gmainar — APPROVED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/loadBalancer.json [hunk 1]
```diff
@@ -550,9 +550,6 @@
           "Update load balancer backend pool with backend addresses containing virtual network and  IP address.": {
             "$ref": "./examples/LBBackendAddressPoolWithBackendAddressesPut.json"
           },
```
Comment (gmainar) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/loadBalancer.json:552]: we may need to remove this comma

**Review by gmainar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**